Buildasaur/ProjectViewController.swift

```diff
@@ -167,7 +167,7 @@ class ProjectViewController: ConfigEditViewController {
         let alreadyHasAuth = auth != nil
 
         switch service {
-        case .GitHub:
+        case .GitHub, .EnterpriseGitHub:
```

Comment:
In light of disallowing OAuth flow for Enterprise, you'll need to add an option here of *only* allowing the token field to be shown (and hide the OAuth login button).